linera-rpc/src/simple/client.rs

```diff
@@ -165,15 +165,15 @@ impl ValidatorNode for SimpleClient {
 #[derive(Clone)]
 pub struct SimpleMassClient {
     pub network: ValidatorPublicNetworkPreConfig<TransportProtocol>,
-    send_timeout: std::time::Duration,
-    recv_timeout: std::time::Duration,
+    send_timeout: linera_base::time::Duration,
+    recv_timeout: linera_base::time::Duration,
 }
 
 impl SimpleMassClient {
     pub fn new(
         network: ValidatorPublicNetworkPreConfig<TransportProtocol>,
-        send_timeout: std::time::Duration,
-        recv_timeout: std::time::Duration,
+        send_timeout: linera_base::time::Duration,
+        recv_timeout: linera_base::time::Duration,
```

Comment:
Optional: same here?

```suggestion
        send_timeout: Duration,
        recv_timeout: Duration,
```